site/kube-addon.md

```diff
@@ -210,7 +278,7 @@ TCP connection from 10.32.0.7:56648 to 10.32.0.11:80 blocked by Weave NPC.
 UDP connection from 10.32.0.7:56648 to 10.32.0.11:80 blocked by Weave NPC.
 ```
 
-### <a name="configuration-options"></a> Changing Configuration Options
+## <a name="configuration-options"></a> Changing Configuration Options
 
 #### Using `cloud.weave.works`
 
```

Comment:
Yes we could link to generic Weave Cloud install instructions, but the "Weave Net Agent" is specifically something we do not want to install at this point.  It's complicated - two different peer discovery models yada yada.